plat/kvm/Config.uk

```diff
@@ -109,8 +112,10 @@ config KVM_VMM_FIRECRACKER
 	select HAVE_MMIO
 	select HAVE_RANDOM
 	select VIRTIO_MMIO_LINUX_COMPAT_CMDLINE
-	imply LIBUKCONSOLE_NS16550 if ARCH_ARM_64
+	imply LIBUKCONSOLE_NS16550 if (ARCH_ARM_64 || ARCH_X86_64)
 	imply LIBUKCONSOLE_NS16550_EARLY_CONSOLE if ARCH_ARM_64
+	imply LIBUKCONSOLE_NS16550_COM1 if ARCH_X86_64
+	imply LIBUKCONSOLE_NS16550_COM_EARLY_CONSOLE if ARCH_X86_64
```

Comment:
Same here, but `LIBUKCONSOLE_NS16550_COM_EARLY_CONSOLE` needs to be renamed to `LIBUKCONSOLE_NS16550_EARLY_CONSOLE`:
```
	imply LIBUKCONSOLE_NS16550 if (ARCH_ARM_64 || ARCH_X86_64)
	imply LIBUKCONSOLE_NS16550_EARLY_CONSOLE
	imply LIBUKCONSOLE_NS16550_COM1 if ARCH_X86_64
```